implementations/duktape/README.md

```diff
@@ -28,6 +28,13 @@ make test
 When you're done building the `nucleus` binary can be used to run and build your
 own tiny JS scripts.
 
+## Testing
+
+Run `make -j4 test`. It will also attempt (re)build if you have not already.
```

Comment:
It attempt to rebuild, so... probably?

That being said, I'm not really sure how parallelized make interacts with rustc. @steveklabnik Any thoughts?